Source/WebCore/page/Frame.cpp

```diff
@@ -257,4 +258,27 @@ void Frame::setOwnerElement(HTMLFrameOwnerElement* element)
     }
 }
 
+void Frame::setOwnerPermissionsPolicy(OwnerPermissionsPolicyData&& ownerPermissionsPolicy)
+{
+    // If the frame is in the same process as parent, it should compute policy from owner element directly.
+    if (dynamicDowncast<RemoteFrame>(tree().parent()))
```

Comment:
Is this check needed? Is it hurting to set the owner policy for local frame as well when getting the info from UIProcess?